src/gd.h

```diff
@@ -627,6 +627,10 @@ BGD_DECLARE(gdImagePtr) gdImageCreateFromWebp (FILE * inFile);
 BGD_DECLARE(gdImagePtr) gdImageCreateFromWebpPtr (int size, void *data);
 BGD_DECLARE(gdImagePtr) gdImageCreateFromWebpCtx (gdIOCtx * infile);
 
+BGD_DECLARE(gdImagePtr) gdImageCreateFromAvif (FILE * inFile);
```

Comment:
i know we've been inconsistent, but we should fix new code at least -- omit the space after the `*`.  you'll want to check all the new lines you add that do this.

Reply:
Got it. Yeah, I couldn't determine the standard from the existing code. Will change my new stuff to omit that space.